Source/DafnyLanguageServer/Workspace/DocumentManager.cs

```diff
@@ -105,15 +109,10 @@ public class DocumentManager {
             relocator.RelocateRange(range, documentChange, CancellationToken.None))).
           Where(r => r != null).Take(MaxRememberedChanges).ToList()!;
     }
-
-    var migratedVerificationTree =
-      relocator.RelocateVerificationTree(lastPublishedState.VerificationTree, updatedText.NumberOfLines, documentChange, CancellationToken.None);
-
     Compilation = new Compilation(
       services,
       verifierOptions,
       updatedText,
-      // TODO do not pass this to CompilationManager but instead use it in FillMissingStateUsingLastPublishedDocument
```

Comment:
I think this TODO still holds. `Compilation` should compile from scratch and be agnostic to anything migration related.